README.md

```diff
@@ -1,106 +1,16 @@
-# openDS
-The home of the specification for open Digital Specimens (openDS).
+open Digital Specimen (openDS) is the cornerstone of DiSSCo's vision for a seamless and standardised digital representation of physical specimens in Natural Science Collections. openDS serves as a comprehensive data specification, unifying diverse scientific information derived from both biological and geological specimens. Born out of the need for a structured description that integrates effortlessly into various workflows, openDS is designed to enhance the Findable, Accessible, Interoperable, and Reusable (FAIR) aspects of biodiversity data.
 
-openDS is still very much in the early stage of development and this repository is where you can follow its progress.
-
-## Getting started
-
-To help you get started, we've provided answers to [frequently asked questions about Digital Specimens and openDS](faqs/faq.md). You can also find there an explanation of [how openDS and the Extended Specimen Network are related](faqs/faqcompare.md), as well as several other faq documents.
-
-The openDS data model relates to other important structures, standards and initiatives in the wider world, as well as to information science in different domains of scientific discourse. [Positioning openDS in the landscape](positioning-opends.md) is one of the first and most important steps in development of the specification. Making sure that everyone likely to make use of the model agrees on this is essential to progress.
-
-openDS consists of three principal and interrelated components, as follows:
-
-- The openDS data model. Read the [introduction to the openDS data model](/data-model/data-model-intro.md);
-- The Ontology for open Digital Specimens (ODS). Read the [introduction to the ODS ontology](/ods-ontology/ods-ont-intro.md); and,
-- The openDS Application Programming Interface (API). Read the [introduction to the openDS API](api-spec/api-intro.md).
-
-## Moving forward
-
-Specification and agreement of openDS is an iterative, spiral process of continuous development (implementation) and improvement (specification) that can begin with the building of one or more 'Hello World' kind of demonstrators. One such example is the demonstrator at [nsidr.org](https://nsidr.org/) / [demo.nsidr.org](https://demo.nsidr.org/). Lessons can be learned from such implementations and fed into the specification process.
+As a digital surrogate on the internet, openDS encapsulates the essence of a "Digital Specimen," offering a solution for collections ranging from marine specimens to botanical and mineralogical holdings. Leveraging established biodiversity information standards such as Darwin Core, Minimal Information Digital Specimen, Latimer Core, Access to Biological Collection Data Schema, Extension for Geosciences, and the emerging Global Biodiversity Information Facility Unified Model, openDS harmonises and elevates the quality and machine actionability of scientific data.
```

Comment:
for in Extension for Geosciences should also be capitalized to: For (as its EFG)

Reply:
fixed